adapter/inventory.go

```diff
@@ -47,3 +47,13 @@ func Inventory() []adapter.RegisterFn {
 		noop.Register,
 	}
 }
+
+// InventoryMap converts adapter inventory to a builder map.
+func InventoryMap(inv []adapter.InfoFn) map[string]*adapter.BuilderInfo {
```

Comment:
Should this function be on BuilderInfoRegistry.go ?

Reply:
I can't find this file. Are you saying we should create a new one ?

Reply:
we can keep that on the validation path.